includes/IslandoraHandleHandleHandler.class.inc

```diff
@@ -67,18 +67,20 @@ abstract class IslandoraHandleHandleHandler {
       // $hostvar will be populated if host resolver is different than ingestion
       // service.
       $hostvar = variable_get('islandora_handle_host', '');
+      // We call the function url with language such that we don't get language specific
+      // prefixes in the URL.
+      // Also, if alias = TRUE it means that the path is used as-is, no alias for the path is used.
```

Comment:
These comments should follow [Drupal standards](https://www.drupal.org/node/1354)  and wrap at 80 characters.

Reply:
I've made the suggested changes.